### ISSUE TITLE
Generate sample Insights data

### ISSUE THREAD

**Issue by mikevespi** [button-inc/emissions-elt-demo#14]
Before we can ingest real data, we need to validate our ingestion and sanitization processes. Thus, we will need to generate sample data that matches the structure of the real data. Ideally the sample data would have values comparable to real values.

The data that we're mocking is the Telus insights data for now. We will likely have the actual ICBC vehicle distributions as they are publicly available.

**Note**
If it is decided that we want mock data for vehicles other than transportation, a new card should be made.

Use a tool like https://json-schema-faker.js.org/ to generate it. We likely want to start somewhere around 500 rows.
You'll need to write the JSON schema forms to use that, but we need them either way.

Definition of Done:
- [ ] Have the sample data in the same shape as the real data will be in
